pages/manual/manual-gettingstarted-build.md

```diff
@@ -100,22 +110,56 @@ Unless you specifically need to debug the application, we recommend setting the
 
 
 ### Mac
-TODO: do
-{% include note.html content="
-When using a multi-configuration generator (like Xcode) you may want to adjust your build mode manually, since it probably defaults to `Debug` at first, which has a large impact on the performance.
-
-If you use a single-configuration generator, you can control the build mode using `CMAKE_BUILD_TYPE` in CMake.
+#### Dependencies
+You will need at least
+- [XCode](https://developer.apple.com/xcode/) 
+- [CMake](https://cmake.org/download/) >= 3.12.0
+- [Qt binaries](https://qt.io/download-open-source/) >= 5.12 
+- [Python](https://www.python.org/downloads/) (optional) is recommended in case you would like to do use Inviwo from Python, write Processors in Python, or perform batch operations. See further (important!) instructions about Python for Mac below.
 
- Unless you specifically need to debug the application, we recommend setting the build mode to `RelWithDebInfo` for good performance, while still getting reasonable stacktraces for debugging and error reporting.
-" %}
+You can use the [brew](https://brew.sh) package manger to install the dependencies using the following commands:
+```
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+brew install --cask cmake
```

Comment:
cmake-gui?

Reply:
Nope, --cask is used for that apparently